### ISSUE TITLE
Data Preprocessing Issue

### ISSUE THREAD

**Issue by LividWo** [chujiezheng/DiffKS#6]
Hi Chujie,

After downloading WOW data using Parlai and unzipping, I got a list of files like 
`
 ["test_random_split.json", "test_topic_split.json", "valid_random_split.json", "valid_topic_split.json", "train.json", ]
`
(which seems like the output of your script  prepare_wow_data.py)

However, the preprocessing script python prepare_wow_data.py is reading files from 
`
d = json.load(open("./Wizard-of-Wikipedia/%s_collected.json" % key, 'r'))
`
which are files with the name like train_collected.json.

I am wondering where are those %s_collected.json files from?

Best,

**Comment by chujiezheng** [comment]
I have uploaded the data [here](https://drive.google.com/drive/folders/1eowwYSfJKaDtYgKHZVqh8alNmqP3jv9A), and this link is also attached in README.

BTW, due to the network problem in China mainland, I used to have trouble in downloading the original data using parlai. The uploaded WoW data were downloaded using the [SLKS codes](https://github.com/bckim92/sequential-knowledge-transformer). And I directly provide the data after preprocessing in the above link. I hope it can facilitate your experiments.
